pkg/ebpf/tracee.go

```diff
@@ -130,6 +130,8 @@ type Tracee struct {
 	// This does not mean they are required for tracee to function.
 	// TODO: remove this in favor of dependency manager nodes
 	requiredKsyms []string
+	// Extensions
+	extensions []Extension
```

Comment:
See my other comment in extenstions.go file - I think we may be able to remove this field from here